src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -465,4 +471,29 @@ public function setResourcesPath($resourcesPath)
     {
         $this->addResourcePath($resourcesPath);
     }
+
+
+    /**
+     * Find out whether to display environment variables from $_ENV
+     *
+     * @return array
+     */
+    public function getEnvVariables()
+    {
+        if( $this->environmentDisplay )
```

Comment:
I presume Graham means "coding style", refering to unusual position of spaces near brackets.